scripts/SANS/sans/gui_logic/presenter/run_tab_presenter.py

```diff
@@ -497,6 +510,7 @@ def update_view_from_table_model(self):
             self._view.add_row(row_entry)
             self._view.change_row_color(row_state_to_colour_mapping[row.row_state], row_index + 1)
             self._view.set_row_tooltip(row.tool_tip, row_index + 1)
+            print(row.tool_tip)
```

Comment:
What is this doing?

Reply:
Looks like something from testing - good spot